src/itemlistformaction.cpp

```diff
@@ -64,15 +64,12 @@ bool ItemListFormAction::process_operation(Operation op,
 	 *   - if an item was selected, then fetch it and do something with it
 	 */
 
-	std::string itemposname = f.get("items_pos");
-	unsigned int itempos = utils::to_u(itemposname);
+	unsigned int itempos = items_list.get_position();
 
 	switch (op) {
 	case OP_OPEN: {
-		LOG(Level::INFO,
-			"ItemListFormAction: opening item at pos `%s'",
-			itemposname);
-		if (itemposname.length() > 0 && visible_items.size() != 0) {
+		LOG(Level::INFO, "ItemListFormAction: opening item at pos `%u'", itempos);
+		if (visible_items.size() != 0) {
```

Comment:
Please change this to use `.empty()` instead, as it's clearer.

Same for all other checks that use `.size() != 0`.